### DISCUSSION TITLE
Create an IKVM Visual Studio e.g. 2022 Extension that translate (e.g. marven) java project to csproj project.

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [ikvmnet/ikvm#152] [Ideas]
# Use Case

Open a folder with marven java project using Visual Studio e.g. 2022 

I understand that this is ongoing progress.

I am suggesting the possibility to be able to port *.java files to *.cs files within Visual Studio 2020

**Comment by wasabii** [comment]
Converting .java source code to C# is out of scope for the IKVM project.

**Comment by GeorgeS2019** [comment]
Of the entire .NET space, ikvm IS THE ONLY ONE that empower .NET developers to work with Java.

Are you aware of Sharpen.jar that translate java to cs?
